org.flozz.calcleaner.yml

```diff
@@ -0,0 +1,31 @@
+app-id: org.flozz.calcleaner
```

Comment:
there is no mention of `flozz.org` on the github repo, and it redirects to an empty placeholder.

does this app-id meet the requirements? Do you (upstream) control it?